src/lib/rest/rest.cpp

```diff
@@ -1128,6 +1128,21 @@ static int connectionTreat
   }
 
   //
+  // Check Content-Type and Content-Length for GET/DELETE requests
+  //
+  if ((ciP->httpHeaders.contentType != "") && (ciP->httpHeaders.contentLength == 0) && ((ciP->verb == GET) || (ciP->verb == DELETE)))
+  {
+    const char*  details = "Content-Type set but no payload supplied in GET/DELETE request";
```

Comment:
Ah, about this text ... 

'GET/DELETE requests cannot have payload, and Content-Type HTTP header is forbidden'  ?

'Orion accepts no payload for GET/DELETE requests - HTTP header Content-Type is thus forbidden' ?